xoinvader/ship.py

```diff
@@ -255,6 +266,22 @@ def __init__(self, pos, border, settings):
         self._fire = True
         self._wbay = Point(x=self._image.width // 2, y=1)
 
+        from xoinvader.animation import AnimationManager
+        self._animgr = AnimationManager()
+        self._animgr.add(
+            name="chaotic", bind=self, attr="direction",
```

Comment:
What is `direction`?
Animation is usually dealing with coordinates, not their deltas.
I think the feature would benefit from example on this too.